Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM python:3.10
+
+RUN apt update
```

Comment:
Combine multiple RUN commands: Instead of having separate RUN commands for updating apt, creating a directory, and upgrading pip, combine them into a single RUN command. This will reduce the number of intermediate layers and make the build process faster.

Example:
RUN apt-get update
&& apt-get install -y --no-install-recommends gcc
&& rm -rf /var/lib/apt/lists/*
&& python -m pip install --upgrade pip \